atlassian_jwt_auth/contrib/flask_app/utils.py

```diff
@@ -0,0 +1,30 @@
+from flask import current_app
+import jwt.exceptions
+import requests.exceptions
+
+
+def parse_jwt(verifier, encoded_jwt):
+    """Decode an encoded JWT using stored config."""
+    try:
+        claims = verifier.verify_jwt(
+            encoded_jwt,
+            current_app.config.asap['VALID_AUDIENCE'],
+            leeway=60,
+            verify=current_app.config.asap.get('VERIFY_TLS_CERT', True)
```

Comment:
Let's remove it and people who need to work with TLS configurations where there isn't a trusted certificate in use will have to write some more code.